docs/source/pillars/computing_technology.md

```diff
@@ -107,6 +107,13 @@ _The ability to run analyses requiring more compute resources, or more specialis
     For example, when using physical compute resources all sensitive data must be securely wiped before another user is given access to that same node.
     In a cloud hosted TRE virtual machines should be destroyed and recreated.
   - Mandatory
+* - A TRE could integrate with large-scale data analytics tools
+  - For example, software such as Spark and Hadoop are used for distributed computing across clusters.
+    Where data included in TREs are impractically large for single-machine computing, this will be an advantage.
+  - Optional
+* - A TRE could integrate with cloud-native managed services
+  - For example, TREs that make use of Spark for large-scale data analytics may also want to use Databricks for automated cluster management.
```

Comment:
Might be useful to have a more broad set of examples _e.g._ "cloud-native services can provide useful tools for researchers such as notebooks, data management, distributed computing, container management" (I'm making this list up, not sure if all of those exist). That way we don't imply it is only useful when a TRE has data analytics tools.

Reply:
I've added an example from #64 (17.3.a) of cloud machine-learning platforms - are there other examples worth mentioning?

Reply:
Just seen Simon's edit